src/main.js

```diff
@@ -133,6 +153,29 @@ function injectRandomPoster() {
   injectPosterValues(currentPoster);
 };
 
+function show(element) {
+  element.classList.remove('hidden')
+};
+
+function hide(element) {
+  element.classList.add('hidden')
+};
```

Comment:
Helper functions! These made all the show section functions much cleaner than I was expecting and are going to keep being helpful moving forward.